frontend/src/pages/Lobbies.tsx

```diff
@@ -1,30 +1,110 @@
 import '../App.css';
-import React from 'react';
 import Header from '../components/header';
+import AllLobbies from '../components/lobbies.json'
+import CreateLobbyForm from '../components/createlobbyform'
+import React, {useEffect, useState } from 'react';
+import { fetchGamesServiceAPI, Game, GamesServiceResponse, getGameImageUrl} from '../utilities';
+import { useAuth } from '../context/AuthContext';
+
+export default function Lobbies() {
+  // State for the games sidebar
+  const {getUser, getAuthToken, setupUser} = useAuth();
+  const [gameResults, setGameResults] = useState<Game[]>([]);
+
+  // Fetches current users games, updates if user changes 
+  useEffect(() => {
+    const fetchGames = async () => {
+      try {
+        const gamesServiceData: GamesServiceResponse | null = await fetchGamesServiceAPI(getAuthToken());
```

Comment:
Reminder to @Alananlan to remove this fetch entirely and just use the stale data from the user obj in AuthContext so we can reduce usage on getOwnedGames API